Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -180,6 +182,11 @@ public async Task<ParticipantStatusCreatedViewModel> UpdateParticipantStatus(str
             await _context.SaveChangesAsync();
             return status;
         }
+
+        public bool IsAdmin(string username)
+        {
+            return _context.Participant.Any(p => p.Username == username && p.IsAdmin == true);
+        }
     }
```

Comment:
You might want to also make a function called IsReferee

Reply:
I have the `IsReferee` in `ActivityService`, because we can't get the ActivityID if we go from `Match` to `Participant`.

Reply:
We may need to figure out a better structure.